python/semantic_kernel/processes/const.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+END_PROCESS_ID: str = "END"
```

Comment:
This has been updated in dotnet to be `"Microsoft.SemanticKernel.Process.EndStep"`